starter-code/ShoppingListWithDetailView/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistdetailview/ShoppingSQLiteOpenHelper.java

```diff
@@ -103,4 +103,31 @@ public long addItem(String name, String description, String price, String type){
 
         return shoppingItems;
     }
+
+    public ShoppingItem getItemId(int id){
```

Comment:
A better name for this method would be getItemById. getItemId makes it sound like you are retrieving the id of an item.